include/standardese/parser.hpp

```diff
@@ -33,7 +33,7 @@ namespace standardese
 
     struct compile_config
     {
-        standardese::cpp_standard cpp_standard;
+        standardese::cpp_standard standard;
```

Comment:
A really minor nitpick: Could you also change `standardese::cpp_standard` to `cpp_standard`?